### ISSUE TITLE
Build should create index.html in the presentation root

### ISSUE THREAD

**Issue by okonet** [Termina1/react-shower#5]
In order to make it work with GitHub hosting (`gh-pages`) and be more compatible with Spectacle I'd propose to put the `index.html` into the presentation's root and not into `/build`. 

Any objections?

**Comment by Termina1** [comment]
actually, I tried to streamline this in reshower (https://github.com/Termina1/reshower), you can do:
`reshower build -d ./`

But, I'm not sure I want to put this by default into the root, since this is an auto generated file.

**Comment by okonet** [comment]
↑ will put all compiled files to the presentation root, which isn't an option.

What about an option in `reshower` that allow you specify where the `index.html` should be created?

**Comment by Termina1** [comment]
it will do this, but only in gh-pages branch, so it doesn't matter

**Comment by okonet** [comment]
That's correct. ATM I'm hosting sources and compiled presentation in `gh-pages` branch for convenience but I should probably stop doing this either way...

**Comment by Termina1** [comment]
We can do build the way, that -d parameter would set index.html path, and everything else will go to /build

**Comment by okonet** [comment]
I actually like this ↑
